Google.PowerShell.IntegrationTests/SQL.One/SQL.GcSqlConfiguration.Tests.ps1

```diff
@@ -25,7 +26,7 @@ Describe "New-GcSqlSettingConfig" {
     It "should be able to instantiate itself with just Tier passed in" {
         $setting = New-GcSqlSettingConfig "D1"
         $setting.Tier | Should Be "D1"
-        $setting.DataDiskSizeGb | Should Be 10
+        $setting.DataDiskSizeGb | Should Be 50
```

Comment:
Sorry about that. I had changed the setting since the default of 10GiB seemed woefully too small, and didn't update the test.